packages/react-charts/src/components/ChartBar/examples/ChartBar.md

```diff
@@ -163,20 +163,105 @@ import { Chart, ChartBar, ChartVoronoiContainer } from '@patternfly/react-charts
 </div>
 ```
 
+### Basic with right aligned legend (Accessibility Demo)
+
+This demo demonstrate the accessibility function in a Bar chart. You can now focus on the whole chart or the contents in the chart. At the same time, you could use screen reader like VoiceOver to read the chart and contents as you defined in format.
+
+```js
+import React from 'react';
+import { Chart, ChartAxis, ChartBar, ChartGroup, ChartVoronoiContainer } from '@patternfly/react-charts';
+
+function BasicWithRightAlignedLegendAccessibilityDemo() {
+  const data = {
+    cat: [
+      { name: 'Cats', x: '2015', y: 1 },
+      { name: 'Cats', x: '2016', y: 2 },
+      { name: 'Cats', x: '2017', y: 5 },
+      { name: 'Cats', x: '2018', y: 3 }
+    ],
+    dog: [
+      { name: 'Dogs', x: '2015', y: 2 },
+      { name: 'Dogs', x: '2016', y: 1 },
+      { name: 'Dogs', x: '2017', y: 7 },
+      { name: 'Dogs', x: '2018', y: 4 }
+    ],
+    birds: [
+      { name: 'Birds', x: '2015', y: 4 },
+      { name: 'Birds', x: '2016', y: 4 },
+      { name: 'Birds', x: '2017', y: 9 },
+      { name: 'Birds', x: '2018', y: 7 }
+    ],
+    mice: [
+      { name: 'Mice', x: '2015', y: 3 },
+      { name: 'Mice', x: '2016', y: 3 },
+      { name: 'Mice', x: '2017', y: 8 },
+      { name: 'Mice', x: '2018', y: 5 }
+    ]
+  };
+  const ariaLabel = Object.assign(
+    ...Object.keys(data).map(type => ({
+      [type]: data[type].map(entry => `There are ${entry.y} ${entry.name} in ${entry.x}`)
+    }))
+  );
+  console.log(ariaLabel);
+  return (
+    <div style={{ height: '250px', width: '600px' }}>
+      <Chart
+        ariaDesc="Average number of pets"
+        ariaTitle="Accessible Bar chart example"
+        containerComponent={
+          <ChartVoronoiContainer
+            role="group"
```

Comment:
It's not clear that `role=group` is required to make tabbing possible. Perhaps this is something we want to set by default?

Reply:
It's `tabIndex` but not `role=group` that makes it possible for us to tab and focus the whole chart. However, it's `role=group` that makes it possible to access the contents inside the chart. If we set it to default `role=img`, we can never tab into the chart and focus on the individual bar.